test/load/metrics.yml

```diff
@@ -1,14 +1,14 @@
-- query: sum by (job) (rate(container_memory_usage_bytes{container="manager", namespace="keptn-lifecycle-toolkit-system"}[1m]))
-  metricName: MemoryUsage
+- query: (max_over_time(container_memory_working_set_bytes{container="manager", namespace="keptn-lifecycle-toolkit-system", pod=~"klc-controller-manager-.*"}[1m]))/1024/1024
+  metricName: MemoryUsageInMB
 
-- query: sum by (job) (rate(container_cpu_usage_seconds_total{container="manager", namespace="keptn-lifecycle-toolkit-system"}[1m]))
+- query: max_over_time(irate(container_cpu_usage_seconds_total{container="manager", namespace="keptn-lifecycle-toolkit-system", pod=~"klc-controller-manager-.*"}[1m])[{{ .elapsed }}:])
```

Comment:
`{{ .elapsed }}` is provided by Kube Burner and contains how long the test run for in minutes. This will be 3min + the time to create the CRDs